api/tests/certification/enrolment/unit/domain/usecases/get-user-certification-eligibility_test.js

```diff
@@ -647,15 +703,169 @@ describe('Certification | Enrolment | Unit | Usecases | get-user-certification-e
                 }),
               );
             });
-          });
-          context('when user has not an acquired certification for this badge', function () {
+          },
+        );
+        context(
+          'when user has a cancelled not rejected for fraud validated pix certification with the required pix score',
```

Comment:
`when user has a validated and cancelled certification that is not rejected for fraud and with the required pixscore `